routes/platform_api.php

```diff
@@ -9,4 +9,6 @@
 
     Route::post('lti/entry', 'PlatformAPI\LtiInterfaceController@entry');
     Route::post('lti/send_result', 'PlatformAPI\LtiInterfaceController@sendResult');
+
+    Route::post('lti_request/send_result', 'PlatformAPI\LtiRequestController@sendResult');
```

Comment:
The goal for the new service is to be as generic as possible for the platform. So that's why I suggested using "send_result" directly on the root of the platform api.

Reply:
platform api is generic for multiple purposes, so controller name used as part of the path

Reply:
So let's call the controller differently then. You can call it "ResultDispatcher" and so have a url `result/send` 
Other proposals are welcome.

Reply:
Still to be done.

Reply:
Controller renamed. Lti prefix still required in controller names and urls, platform API works not only with lti.

Reply:
My request for something generic was on purpose to abstract this from LTI as the caller does not care that the publish is done for LTI or something else.